php/WP_CLI/CommandWithUpgrade.php

```diff
@@ -264,18 +264,24 @@ protected function update_many( $args, $assoc_args ) {
 			\WP_CLI::error( $line );
 		}
 
-		if ($num_to_update > 0) {
-			$status = array();
-			foreach($items_to_update as $item_to_update => $info) {
-				$status[$item_to_update] = array(
-					'name' => $info['name'],
-					'old_version' => $info['version'],
-					'new_version' => $info['update_version'],
-					'status' => $result[$item_to_update] !== null ? 'Updated' : 'Error',
-				);
+		if ( $num_to_update > 0 ) {
+			if ( ! empty( $assoc_args['format'] ) && 'summary' === $assoc_args['format'] ) {
+				foreach( $items_to_update as $item_to_update => $info ) {
+					$message = $result[$item_to_update] !== null ? 'updated successfully' : 'did not update';
```

Comment:
`$items_to_update` has the value of
```php
array(1) {
  ["/var/www/subdirwp/server/static/themes/p2"]=>
  array(9) {
    ["name"]=>
    string(2) "p2"
    ["status"]=>
    string(8) "inactive"
    ["update"]=>
    bool(true)
    ["update_version"]=>
    string(5) "1.5.5"
    ["update_package"]=>
    string(50) "https://downloads.wordpress.org/theme/p2.1.5.5.zip"
    ["version"]=>
    string(5) "1.4.2"
    ["update_id"]=>
    string(2) "p2"
    ["title"]=>
    string(2) "P2"
    ["description"]=>
```
MAybe `$message = $result[ $info['name'] ]`?